editor/settings/editor_settings.cpp

```diff
@@ -440,6 +440,9 @@ void EditorSettings::_load_defaults(Ref<ConfigFile> p_extra_config) {
 	EDITOR_SETTING(Variant::BOOL, PROPERTY_HINT_NONE, "interface/editor/localize_settings", true, "")
 	EDITOR_SETTING_BASIC(Variant::INT, PROPERTY_HINT_ENUM, "interface/editor/dock_tab_style", 0, "Text Only,Icon Only,Text and Icon")
 	EDITOR_SETTING_USAGE(Variant::INT, PROPERTY_HINT_ENUM, "interface/editor/ui_layout_direction", 0, "Based on Application Locale,Left-to-Right,Right-to-Left,Based on System Locale", PROPERTY_USAGE_DEFAULT | PROPERTY_USAGE_RESTART_IF_CHANGED)
+	EDITOR_SETTING(Variant::PACKED_STRING_ARRAY, PROPERTY_HINT_NONE, "interface/editor/favorite_language_codes", PackedStringArray(), "");
+	EDITOR_SETTING(Variant::PACKED_STRING_ARRAY, PROPERTY_HINT_NONE, "interface/editor/favorite_script_codes", PackedStringArray(), "");
+	EDITOR_SETTING(Variant::PACKED_STRING_ARRAY, PROPERTY_HINT_NONE, "interface/editor/favorite_country_codes", PackedStringArray(), "");
```

Comment:
Is it any useful to show these settings in the dialog?
If not, they can be changed to begin with underscore (`_favorite_language_codes` etc.), so they are not "exposed".